cartography/intel/gitlab/projects.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+import requests
+
+from cartography.util import make_requests_url
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_projects(access_token:str,group:str):
+    url = f"https://gitlab.example.com/api/v4/groups/{group}/projects?per_page=100"
```

Comment:
same, add documentation url in the source code as a comment.

Reply:
yes, added